### ISSUE TITLE
Category page header hierarchy

### ISSUE THREAD

**Issue by phette23** [cca/libraries_wagtail#177]
https://libraries.cca.edu/services/

- [ ] lacks h1
- [ ] everything is an h2; should be h2 around row title and then child elements are h3
